### ISSUE TITLE
Delete unnecessary folders/files in Nestor Branch

### ISSUE THREAD

**Issue by danxzylo** [danxzylo/JavaIISearchEngine#5]
Been checking for bugs in the Nestor branch, everything seems to be in working order. I'd like for Nestor to delete the extra folders not needed for the project and push those changes to the Nestor branch, so it can be merged in to the SearchEngineMaster branch.

The file/folder(s) we might not need are:

SearchEngine.java
SearchEngine/src/searchengine (Ricci's old project)
JavaProject2 (It looks like this is where the index file is saved. If it's auto generated by the program we probably don't need it on git but leave it alone if it's needed or can't be removed)

Thanks,
-Danny

**Comment by danxzylo** [comment]
Once this is done, make a pull request and let me know so we can complete the merge.
